src/graphics/program-lib/programs/standard.js

```diff
@@ -588,9 +588,9 @@ const standard = {
         if (options.cubeMap) options.sphereMap = null; // cubeMaps have higher priority
         if (options.dpAtlas) options.prefilteredCubemap = null; // dp has even higher priority
         if (!options.useSpecular) options.specularMap = options.glossMap = null;
-        const needsNormal = lighting || reflections || options.ambientSH || options.prefilteredCubemap || options.heightMap || options.enableGGXSpecular ||
-                            LayerComposition.clusteredLightingEnabled || options.clearCoatNormalMap;
         const shadowPass = options.pass >= SHADER_SHADOW && options.pass <= 17;
+        const needsNormal = lighting || reflections || options.ambientSH || options.prefilteredCubemap || options.heightMap || options.enableGGXSpecular ||
+                            (LayerComposition.clusteredLightingEnabled && !shadowPass) || options.clearCoatNormalMap;
```

Comment:
@mvaligursky does `!shadowPass` only apply to clustered? surely this applies everywhere?

Reply:
this bit applies only to clustered lighting, and that's what I changed. Have I missed something?

`(LayerComposition.clusteredLightingEnabled && !shadowPass)`

What it says is that the shader needs normal in case its clustered shader (in addition to all existing conditions) but not rendering to shadow map.

Reply:
Sorry what I mean is: do we ever need normals in a shadow pass? Seems to me, whether rendering clustered or not we probably don't want normals in a shadow pass!

Reply:
ah I see what you mean. Normally shadow passes don't have normals, this is gated by 
`let lighting = options.lights.length > 0;`

so the addition of `(LayerComposition.clusteredLightingEnabled && !shadowPass)` is more about adding normals when clustered lighting needs it, rather than removing it for other cases - as when clustered lights are on, the options.lights.length is 0 and so normals would be missing.